### PR TITLE
Update to the macOS guide

### PR DESCRIPTION
**Pull request by DubhAd**
1. Updated the Python version
2. Changed it to a venv install to avoid people breaking things by accident
3. Linked to the auto-start guide
4. Removed link to the outdated video

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/installation/macos.markdown [hunk 1]
```diff
@@ -11,12 +11,15 @@ footer: true
...
 [macOS](http://www.apple.com/macos/) is available by default on Apple computer. If you run a different operating system, please refer to the other section of the documentation.
 
-To run Home Assistant on macOS you need to install Python first. Download Python 3.5.3 or later from [https://www.python.org/downloads/mac-osx/](https://www.python.org/downloads/mac-osx/) and follow the instructions of the installer.
+To run Home Assistant on macOS you need to install Python first. Download Python 3.6 or later (at this time we recommend Python 3.7) from [https://www.python.org/downloads/mac-osx/](https://www.python.org/downloads/mac-osx/) and follow the instructions of the installer.
 
-Open a terminal and install Home Assistant.
+Open a terminal and install Home Assistant in a virtual environment:
 
 ```bash
+$ pip3 install virtualenv
```
Comment (frenck) [inline source/_docs/installation/macos.markdown]: Why `virtualenv` and not the build-in `venv`?

File: source/_docs/installation/macos.markdown [hunk 2]
```diff
@@ -11,12 +11,15 @@ footer: true
 
 [macOS](http://www.apple.com/macos/) is available by default on Apple computer. If you run a different operating system, please refer to the other section of the documentation.
 
-To run Home Assistant on macOS you need to install Python first. Download Python 3.5.3 or later from [https://www.python.org/downloads/mac-osx/](https://www.python.org/downloads/mac-osx/) and follow the instructions of the installer.
+To run Home Assistant on macOS you need to install Python first. Download Python 3.6 or later (at this time we recommend Python 3.7) from [https://www.python.org/downloads/mac-osx/](https://www.python.org/downloads/mac-osx/) and follow the instructions of the installer.
```
Comment (frenck) [inline source/_docs/installation/macos.markdown]: Maybe we should switch to `brew` for this? Dunno... ?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/installation/macos.markdown (hunk 2 above)
Comment (DubhAd) [reply source/_docs/installation/macos.markdown]: If you can provide a pointer to how - I've not touched a Mac in more years than I care to count

File: source/_docs/installation/macos.markdown (hunk 1 above)
Comment (DubhAd) [reply source/_docs/installation/macos.markdown]: Good question, I deleted the wrong line 🤦‍♂

Comment (frenck) [reply source/_docs/installation/macos.markdown]: Addressed

File: source/_docs/installation/macos.markdown (hunk 2 above)
Comment (frenck) [reply source/_docs/installation/macos.markdown]: Left it out for now. Since it would require details on how to setup Brew as well.

**Review by frenck — APPROVED** [review]

**Pull request merged**